examples/kuramotosivashinksy/analysis.jl

```diff
@@ -8,21 +8,20 @@ using Plots
 using Random
 using Statistics
 using TOML
+using HDF5
 
 using CliMAgen
 package_dir = pkgdir(CliMAgen)
 include(joinpath(package_dir,"examples/utils_data.jl"))
 include(joinpath(package_dir,"examples/utils_analysis.jl"))
-include(joinpath(package_dir,"examples/utils_etl.jl"))
-
-function run_analysis(params; FT=Float32, logger=nothing)
 
+function run_analysis(params; FT=Float32, k_bias=0.0f0, n_avg=1)
```

Comment:
this script now never makes samples, so `make_samples` is not longer in the experiment toml.